EXM_105/Makefile

```diff
@@ -1,56 +1,13 @@
-all: info .setup-cqf-ruler synthea generate-patients calculate-patients
+all stu3: info generate-patients-stu3 calculate-patients-stu3
 
 info:
-	$(info `make` will cache the cqf-ruler, connectathon, and synythea setups, but repeat patient generation and calculation.)
-	$(info To repeat all steps, do `make clean; make`)
-
-.setup-cqf-ruler: .cqf-ruler connectathon .seed-cqf-ruler
-	touch .setup-cqf-ruler
-
-.cqf-ruler:
-	docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler:develop
-	touch .cqf-ruler
-
-connectathon:
-	git clone https://github.com/DBCG/connectathon.git
-
-.seed-cqf-ruler:
-	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
-	curl -X POST \
-  http://localhost:8080/cqf-ruler-dstu3/fhir \
-  -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-bundle.json
-	touch .seed-cqf-ruler
-
-.seed-cqf-ruler-no-vs:
-	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
-	curl -X POST \
-  http://localhost:8080/cqf-ruler-dstu3/fhir \
-  -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-files/library-deps-EXM105_FHIR3-8.0.000-bundle.json
-	curl -X PUT \
-  http://localhost:8080/cqf-ruler-dstu3/fhir/Library/library-EXM105-FHIR3-8.0.000 \
-  -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-files/library-EXM105_FHIR3-8.0.000.json
-	curl -X PUT \
-  http://localhost:8080/cqf-ruler-dstu3/fhir/Measure/measure-EXM105-FHIR3-8.0.000 \
-  -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-files/measure-EXM105_FHIR3-8.0.000.json
-	touch .seed-cqf-ruler-no-vs
-
-synthea:
-	git clone --single-branch --branch abacus --depth 1 https://github.com/projecttacoma/synthea.git
+	$(info `make` will perform patient generation and calculation.)
 
 PATIENT_COUNT := 30
-generate-patients:
-	cd synthea && ./run_synthea -a 18-21 -p $(PATIENT_COUNT) -m stroke_exm_105
-
-CQL_FILE := connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-files/EXM105_FHIR3-8.0.000.cql
-calculate-patients:
-	calculate-bundles -d ./synthea/output/fhir_stu3 -c $(CQL_FILE) -u http://localhost:8080/cqf-ruler-dstu3/fhir --measure-id measure-EXM105-FHIR3-8.0.000
-
-clean:
-	-docker stop cqf-ruler
-	-rm -rf synthea output connectathon .cqf-ruler .seed-cqf-ruler .seed-cqf-ruler-no-vs .generate-patients .calculate-patients .setup-cqf-ruler
+generate-patients-stu3:
```

Comment:
missing generate-patients-r4 for this Makefile for some reason (both before and after these changes)

Reply:
Yeah i guess we just haven't yet done R4 for this measure. Probably out of scope of this task as well though.

Reply:
Sounds good--that will be a pre-req for the task to publish a new pre-loaded r4 docker image

Reply:
created https://jira.mitre.org/browse/TACOSTRAT-396